lusSTR/filters.json

```diff
@@ -0,0 +1,488 @@
+{
+    "CSF1PO": {
+        "MinimumNumberReadsForDynamicThresholds": 650,
+        "DetectionThresholdStaticCount": 10,
+        "DetectionThresholdDynamicPercent": 0,
+        "DetectionThresholdUse": "Static",
+        "AnalyticalThresholdStaticCount": 20,
```

Comment:
Did you intend to include two copies of this file?

Reply:
lol nope. I don't even know how I did that.